src/components/tx/modals/NewTxModal/index.tsx

```diff
@@ -50,14 +59,21 @@ const NewTxModal = ({ onClose, recipient }: { onClose: () => void; recipient?: s
     setNftModalOpen(true)
   }
 
+  const onRejectModalOpen = () => {
+    trackEvent(MODALS_EVENTS.REJECT_TX)
+    setRejectModalOpen(true)
+  }
+
   const onContractInteraction = () => {
     trackEvent(MODALS_EVENTS.CONTRACT_INTERACTION)
     onClose()
   }
 
+  const dialogTitle = txNonce ? `Replace transaction #${txNonce}` : 'New transaction'
+
   return (
     <>
-      <ModalDialog open dialogTitle="New transaction" onClose={onClose}>
+      <ModalDialog open dialogTitle={dialogTitle} onClose={onClose}>
         <DialogContent>
           <Box display="flex" flexDirection="column" alignItems="center" gap={2} pt={7} pb={4} width={240} m="auto">
```

Comment:
Should we add some info-text here if the nonce is set explaining how replacement works?

Something along the lines of `Replacing a transaction will create a transaction with  the same nonce. To actually replace / reject this new transaction has to get executed to use that nonce`

Reply:
It looked somewhat odd with text inside the modal. I've added a tooltip. What do you think?

![tooltip](https://user-images.githubusercontent.com/20442784/199947953-52ee6dd4-b9e7-4723-9eba-616bd0781308.gif)

Reply:
Nice! I like it.